### ISSUE TITLE
Incorrect parameters declaration breaks apigee_edge module (PHP 8.0)

### ISSUE THREAD

**Issue by VladimirAus** [apigee/apigee-client-php#196]
## Issue

* This is not full support for PHP8.0 issue but rather bug about incorrectly declared parameters in the library.
* `PHP 8.0` and `8.1` are currently not supported by the module. 
* `Drupal 10` (`alpha1` is available now) will have minimum `PHP 8.0` or `8.1` requirement.
* Corresponding module issue: https://github.com/apigee/apigee-edge-drupal/issues/657

### Teams / Team Apps page:
```
TypeError: Apigee\Edge\Api\Management\Entity\AppOwner::setApps(): Argument #1 must be of 
type string, array given, called in /root/vendor/symfony/property-access/PropertyAccessor.php 
on line 570 in Apigee\Edge\Api\Management\Entity\AppOwner->setApps() (line 58 of 
/root/vendor/apigee/apigee-client-php/src/Entity/Property/AppsPropertyAwareTrait.php)
```

```
Message | TypeError: Apigee\Edge\Api\Management\Entity\AppCredential::setApiProducts(): 
Argument #1 must be of type Apigee\Edge\Structure\CredentialProductInterface, array given, 
called in /root/vendor/symfony/property-access/PropertyAccessor.php on line 570 
in Apigee\Edge\Api\Management\Entity\AppCredential->setApiProducts() (line 100 of 
/root/vendor/apigee/apigee-client-php/src/Api/Management/Entity/AppCredential.php)
```

```
TypeError: Apigee\Edge\Api\Management\Entity\App::setCredentials(): Argument #1 must be of 
type Apigee\Edge\Api\Management\Entity\AppCredentialInterface, array given, called in 
/root/vendor/symfony/property-access/PropertyAccessor.php on line 570 in 
Apigee\Edge\Api\Management\Entity\App->setCredentials() (line 199 of 
/root/vendor/apigee/apigee-client-php/src/Api/Management/Entity/App.php) 
```

### Status Report page:
```
TypeError: Apigee\Edge\Api\Management\Entity\Organization::setEnvironments(): Argument #1 
must be of type string, array given, called in /root/vendor/symfony/property-access/PropertyAccessor.php 
on line 570 in Apigee\Edge\Api\Management\Entity\Organization->setEnvironments() (line 42 of 
/root/vendor/apigee/apigee-client-php/src/Entity/Property/EnvironmentsPropertyAwareTrait.php) 
```

**Comment by mourad-zitouni** [comment]
I'm using Apigee with drupal, I have the same issue with PHP8

**Comment by divya-intelli** [comment]
Hi @VladimirAus , @mourad-zitouni this issue is in progress and is being worked on by our team. Will keep you posted here. Thank you.

**Comment by mourad-zitouni** [comment]
Hi @divya-intelli , thanks a lot for your reply!

**Comment by patricia-sevilla** [comment]
Hello, I also have the same problem.
I will be attentive to the news.
Thank you very much

**Comment by divya-intelli** [comment]
Hi,  apigee-client-php is compatible with PHP 8.0 on dev version. 
We will update once apigee-client-php library is compatible with PHP 8.1.

**Comment by VladimirAus** [comment]
😭

**Comment by divya-intelli** [comment]
@VladimirAus @mourad-zitouni @patricia-sevilla , we have released apigee-edge-drupal 2.0.2 which supports PHP 8.1.
